src/components/Reviews.jsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from "react"
+
+function Reviews() {
+    const [reviewList, setReviewList] = useState([]);
+
+    useEffect(() => {
+        fetch("https://nc-games-sarah-hume.onrender.com/api/reviews")
```

Comment:
Fetching should be done in a separate file that can be called api.js, here you would instead invoke the function imported from said file 🙂